server/index.js

```diff
@@ -5,12 +5,14 @@ var registerUserHandler = require("./registerUserHandler");
 var listAttractionsHandler = require("./listAttractionsHandler");
 var attractionDetailHandler = require("./attractionDetailHandler");
 var authenticateHandler = require("./authenticateHandler");
+var cartHandler = require("./cartHandler");
 
 var handle = {};
 handle["/register"] = registerUserHandler.register;
 handle["/attractions"] = listAttractionsHandler.getAttractions;
 handle["/attractiondetails"] = attractionDetailHandler.getAttractionDetails;
 handle["/authenticate"] = authenticateHandler.authenticate;
+handle["/validateToken"] = cartHandler.validateToken;
```

Comment:
Don't map token validator to any URL. We are not going to expose this as the service.